src/components/Export.tsx

```diff
@@ -115,10 +117,9 @@ export default function Export(inputProps: ExportButtonProps): ReactElement {
   // Note: notification API seems to only place notifications at the top-level under the
   // <body> tag, which causes some issues with styling.
   const { modal, notification } = App.useApp();
-  const modalContextRef = useRef<HTMLDivElement>(null);
 
   const originalFrameRef = useRef(props.currentFrame);
-  const [isLoadModalOpen, _setIsLoadModalOpen] = useState(false);
+  const [isModalOpen, _setIsModalOpen] = useState(false);
```

Comment:
Rename here, since `isLoadModalOpen` was a holdover from when I copied this code from `LoadButton.tsx`.